mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -1736,7 +1745,9 @@ func flattenComposerEnvironmentConfigWorkloadsConfig(workloadsConfig *composer.W
 	transformed["web_server"] = []interface{}{transformedWebServer}
 	transformed["worker"] = []interface{}{transformedWorker}
 <% unless version == "ga" -%>
-	transformed["dag_processor"] = []interface{}{transformedDagProcessor}
+	if transformedDagProcessor != nil {
```

Comment:
What behavior are this the changes for EnablePrivateEnvironment and EnablePrivateBuildsOnly fixing?

Reply:
I added this to prevent an empty dag_processor in the output of 'terraform state show'. This maintains a cleaner state for composer 2

Reply:
I'm not able to reproduce any issue with `terraform state show` with the released provider; however, as far as I can tell, this seems like a no-op from terraform's perspective, since an empty list is the zero value of a list.